migtests/scripts/live-migration-fallb-run-test.sh

```diff
@@ -67,7 +67,10 @@ main() {
 
 	step "Assess Migration"
 	if [ "${SOURCE_DB_TYPE}" = "postgresql" ]; then
-		assess_migration
+		assess_migration || {
+			cat_log_file "yb-voyager-assess-migration.log"
+			cat_file ${EXPORT_DIR}/assessment/metadata/yb-voyager-assessment.log
```

Comment:
why is the log file under `assessment/metadata` ?

Reply:
I see, I get it now. You only have access to the assessment metadir from the assessment script. That makes sense 👍  . 
Nit: maybe put it inside a /log dir?

Reply:
yeah, but one more `/log` dir will increase the nesting, i thought to keep it outside so that its visible to users upfront. Plus we wont have more than one log file, same is appended everytime...

Reply:
sounds good